#!/usr/bin/env python
PACKAGE = "bicycle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("LQR_Switch", bool_t, 0, "LQR Parameter",    False)
gen.add("LQR_Kp", double_t, 0, "LQR Parameter",    1, 0,   50)
gen.add("LQR_Kv", double_t, 0, "LQR Parameter",    1, 0,   50)
gen.add("LQR_Ks", double_t, 0, "LQR Parameter",    0.0102,0,   10)

exit(gen.generate(PACKAGE, "dynamic_LQR", "LQR"))